### PR TITLE
Random Nice Dogs

### PR DESCRIPTION
**Pull request by jtexeira**
CAO AU AU

### CODE REVIEW

**Review by JoseFilipeFerreira — CHANGES_REQUESTED** [review]
Please move this to a separate command

**Comment by jtexeira** [comment]
@JoseFilipeFerreira Created a new file with the command, since the command didn't fit in any other file

File: bot.py [hunk 1]
```diff
@@ -98,7 +102,7 @@ def main():
     if (message.content.lower() == 'push %ebp'):
         await bot.send_message(message.channel, 'pop %recurso')
 
-    if message.content.startswith(bot.command_prefix):
+    if message.content.startswith('*'):
```
Comment (JoseFilipeFerreira) [inline bot.py]: replace it by `bot.command_prefix` like it was before

File: bot.py [hunk 2]
```diff
@@ -33,10 +36,11 @@ def main():
     bot.IMAGES_PATH = './Media/Images/'
     bot.GIFS_PATH = './Media/Gif/'
     bot.MUSIC_PATH = './Media/Music/'
-    bot.TMP_PATH = './Media/Tmp/'
+    bot.MEMEGENERATOR_PATH = './Media/Memegenerator/'
```
Comment (JoseFilipeFerreira) [inline bot.py]: why did you readd the path for a non-existing path?

File: bot.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 import os
 from os import path
 import subprocess
+import requests
```
Comment (JoseFilipeFerreira) [inline bot.py]: I believe that this is no longer nedded in this file

**Review by JoseFilipeFerreira — CHANGES_REQUESTED** [review]

File: bot.py [hunk 4]
```diff
@@ -40,7 +40,6 @@ def main():
     bot.BATTLEROYALE_PATH = './modules/battleroyale.json'
     bot.BATTLEROYALEWINS_PATH = './modules/battleroyalewins.json'
     bot.EXTENSIONS_PATH ='Extensions.'
-    bot.TMP_PATH = './Media/Tmp/'
```
Comment (JoseFilipeFerreira) [inline bot.py]: The one that was nedded was this one not the Memegenerator

**Review by JoseFilipeFerreira — CHANGES_REQUESTED** [review]

File: Extensions/dogs.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+        while isVideo:
+            r = requests.get('https://random.dog/woof.json')
+            js = r.json()
+            if js['url'].endswith('.mp4'):
+                pass
+            else:
+                isVideo = False
+                colours = [0x1abc9c, 0x11806a, 0x2ecc71, 0x1f8b4c, 0x3498db, 0x206694, 0x9b59b6, 0x71368a, 0xe91e63, 0xad1457, 0xf1c40f, 0xc27c0e, 0xa84300, 0xe74c3c, 0x992d22, 0x95a5a6, 0x607d8b, 0x979c9f, 0x546e7a]
+                col = int(random.random() * len(colours))
+                em = discord.Embed(color=colours[col])
```
Comment (JoseFilipeFerreira) [inline Extensions/dogs.py]: could replace this with choice(colours)

**Review by JoseFilipeFerreira — CHANGES_REQUESTED** [review]

**Review by JoseFilipeFerreira — APPROVED** [review]

**Pull request merged**